webapp/app.py

```diff
@@ -15,13 +18,19 @@
 
 
 class WebApp:
-    def __init__(self, resources_path):
+
```

Comment:
remove extra newline ?

Reply:
Was intentional - but has been [removed](https://github.com/symbol/nodewatch/pull/3/commits/1f283f8b95ef642295574c33e0b11519ead40557) to conform with the maintainer's stylistic preferences.